apps/nextjs-website/src/app/layout.tsx

```diff
@@ -73,6 +73,29 @@ export default async function RootLayout({
             dangerouslySetInnerHTML={{ __html: MATOMO_SCRIPT }}
           />
         )}
+        <meta property='og:title' content='PagoPA DevPortal' />
+        <meta property='og:type' content='website' />
+        <meta property='og:image:width' content='1140' />
+        <meta property='og:image:height' content='600' />
+        <meta property='og:url' content='https://developer.pagopa.it' />
+        <meta
+          property='og:image'
+          content='https://developer.pagopa.it/images/dev-portal-home.jpg'
+        />
+        twitter
+        <meta name='twitter:card' content='summary_large_image' />
+        <meta name='twitter:site' content='@pagopa' />
+        <meta name='twitter:title' content='PagoPA DevPortal' />
+        <meta
+          name='twitter:description'
+          content='Il portale per gli sviluppatori di PagoPA'
+        />
+        <meta
+          name='twitter:image'
+          content='https://developer.pagopa.it/images/dev-portal-home.jpg'
+        />
+        <meta name='twitter:image:alt' content='PagoPA DevPortal' />
+        <meta name='twitter:creator' content='@pagopa' />
```

Comment:
By doing this you are going to add these tags to all the pages using the layout and duplicate page-specific meta tags created with the `makeMetadata` helper (see `apps/nextjs-website/src/helpers/metadata.helpers.ts`).
You should use the helper's function in `app/page.tsx` to set the tags in the home page